core/src/main/java/com/rudderstack/android/sdk/core/RudderContext.java

```diff
@@ -30,6 +32,8 @@
     private RudderNetwork networkInfo;
     @SerializedName("timezone")
     private String timezone = Utils.getTimeZone();
+    @SerializedName("anonymousId")
```

Comment:
We put `anonymousId` at the top level of Message. This doesn't need to be serialized

Reply:
done